app/resources/numismatic_finds/numismatic_find.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class NumismaticFind < Resource
+  include Valkyrie::Resource::AccessControls
+  attribute :member_ids, Valkyrie::Types::Array
+
+  attribute :place
+  attribute :date
+  attribute :find_number, Valkyrie::Types::Anything
```

Comment:
Please make this a `Valkyrie::Types::Int` to make sure it's stored as a number instead of as a string, to make querying for it easier.